dsl/common/src/main/groovy/net/neoforged/gradle/dsl/common/util/Constants.groovy

```diff
@@ -27,7 +27,7 @@ class Constants {
     public static final String DEFAULT_PARCHMENT_GROUP = "org.parchmentmc.data"
     public static final String DEFAULT_PARCHMENT_ARTIFACT_PREFIX = "parchment-"
     public static final String DEFAULT_PARCHMENT_MAVEN_URL = "https://maven.parchmentmc.org/"
-    public static final String JST_TOOL_ARTIFACT = "net.neoforged.jst:jst-cli-bundle:1.0.35"
+    public static final String JST_TOOL_ARTIFACT = "net.neoforged.jst:jst-cli-bundle:1.0.36"
```

Comment:
I would make this globally configurable instead of just in parchment.

Reply:
What do you mean?

Reply:
The Parchment subsystem can configure the JST artifact (and therefore version) used to apply parchment. The JST version used by ATs should also be configurable (mostly for testing).